content/en/getting_started/synthetics/browser_test.md

```diff
@@ -19,77 +22,117 @@ further_reading:
-The browser test details page includes details about your test configuration, test uptime, historical graphs for response time and time to interactive for the first page, sample successful and failed results, and a list of test results corresponding to the selected timeframe. Each individual test result includes screenshots, core web vitals, potential errors, resources, and traces for each step.
+You can wait for your test to generate results in **Test Results** or click **Run test now** on the top right to trigger results quickly.
 
-Wait for your test to generate several test results or hit `Run test now` to trigger them more quickly. Then look for a failed test result under **Test Results** or in your mailbox. You can start your troubleshooting by looking at the screenshots to try to understand what went wrong. Don't forget to look at screenshots of steps that happened before the failed step as these often contain the root cause of the failure.
+To troubleshoot a failed test, scroll down to **Test Results** and click on a failed test result. Review screenshots, core web vitals, potential errors, resources, and traces for each step to diagnose the issue. 
 
-The **Errors & Warnings** tab provides a list of Javascript and network errors, the **Resources** tab locates the resource providing this status, and the **Traces** tab maps the entirety of the request in seconds. This test failed as the result of a server timeout. The resource, `https://api.shopist.io/checkout.json`, posted the status ,and the targeted source of the problem is a controller linked to checkout. You have now successfully found the route of the problem.
+When you look at screenshots leading up to the failed step, you can identify the root cause of the failure more easily. Click on resources, errors, and traces that appear next to the duration graph to access **Errors & Warnings**, **Resources**, and **Traces**. 
+
+**Errors and Warnings** display JavaScript and network errors, **Resources** display the resource providing this status, and **Traces** map the entirety of the request in seconds. 
+
+In the example below, the test failed as the result of a server timeout.
 
 {{< img src="getting_started/synthetics/browser-test-failure.png" alt="Browser test failure"  style="width:100%;">}}
 
-The **Traces** tab is accessible with Datadog [APM integration with Synthetic Monitoring][8]. Once configured, it allows you to go from a test run that failed to the root cause of the issue by looking at the trace generated by that test run. To link browser test results with APM, whitelist the URLs you want the APM integration headers added to. Use `*` for wildcards: `https://*.datadoghq.com/*`
+The resource, `https://api.shopist.io/checkout.json`, posted a status and the target source of this issue is a controller linked to checkout.
+
+With Datadog's [APM integration with Synthetic Monitoring][8], you can access the root cause of a failed test run with a trace generated from the test run. 
```

Comment:
I think you removed or changed some key phrases here that were better before, specifically about the Traces tab, and "it allows you to go from a test run that failed to the root cause of the issue by looking at the trace generated by that test run."